prime-dotnet-webapi/Controllers/JobsController.cs

```diff
@@ -77,5 +80,20 @@ public async Task<ActionResult> PopulateTransactionLogTemp(int numberOfDays)
             var result = await _reportingService.PopulateTransactionLogTempAsync(numberOfDays);
             return Ok(result);
         }
+
+        // POST: api/jobs/populate/organization-registration-id
+        /// <summary>
+        /// execute job to update the contact information in practitioner table by calling PharmaNet API
```

Comment:
Comment needs updating

Reply:
Update comment and add comment to other function as well.